sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/join/CoGroupByKey.java

```diff
@@ -74,6 +75,16 @@
 public class CoGroupByKey<K> extends
     PTransform<KeyedPCollectionTuple<K>,
                PCollection<KV<K, CoGbkResult>>> {
+
+  /**
+   *  options to filter output results.
+   *
+   */
+  public enum FilterOption {
```

Comment:
only need to expose for test, may be defined as 
`@VisibleForTesting static enum FilterOption`